CONTRIBUTING.md

```diff
@@ -44,6 +44,8 @@ Tokens help us achieve simpler customization for complex higher order components
 
 This section covers creating and adding a new component package to FluentUI React Native's monorepo. If you are instead working on an existing component and adding a native module, skip to the next two sections.
 
+Most components should use the compose framework as it offers the comprehensive set of patterns like tokens and slots, but if you're creating a simple component that doesn't require those patterns, there's a lighter pattern called stagedComponent. The stagedComponent pattern splits up the render function into two stages. Stage 1 handles building props and hook calls (best to separate the hook calls from the render tree since they rely on call order). Stage 2 returns the actual element tree, any conditional branching should happen here (Icon is a good example of using stagedCompoenent).
+
```

Comment:
Thanks, this actually helped me understand it too =) 
This actually filled a niche I was thinking about: How do we add theming to a simple existing component like ActivityIndicator. Might be a future hackathon idea to make some simple component with this API. Def not part of the review though, just thoughts

Reply:
Not sure if it belongs in documentation, but could we maybe drop a link to the actual implementation? https://github.com/microsoft/fluentui-react-native/blob/3c0f5496f0daeb748c6b2daac8d0465a1d58a672/packages/experimental/use-slots/src/stagedComponent.ts